File: 51nod/1995.cpp
#include <iostream>
#include <stdio.h>
using namespace std;

int main() {
	int t, x, y;
	cin >> t;
	while(t--) {
		cin >> x >> y;
		if((x==1 or x== 3) and (y==1 or y ==4)) printf("Win\n6\n");
		else if(x==2 and (y==1 or y == 4)) printf("Equal\n0\n");
		else printf("Win\n4\n");
	}
	return 0;
}

File: 51nod/1995.py
t = int(input())
for i in range(t):
    x,y = map(int, input().split())
    if (x==1 or x==3 ) and (y==1 or y == 4):
        print("Win\n6")
    elif x==2 and (y==1 or y==4):
        print("Equal\n0")
    else:
        print("Win\n4")